package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
```

Comment:
- [ ] After following the instructions about how to run the app successful in the README file, here is the result as seen in the screenshot
![Screenshot from 2022-10-18 06-22-28](https://user-images.githubusercontent.com/46388113/196352216-ac09e658-3421-432d-8db9-7ae33c209d3b.png)

This error clearly points us to webpack configuration. Let us kindly find and fix it. Here is the [link](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/lessons/rails_and_webpack.md) we can reference